### PR TITLE
Save token within sharing tab for now

### PR DESCRIPTION
**Pull request by dvargasfuertes**
The 3 users who have static site extension will be accessing it using their legacy roamjs tokens. Long term, we want everything through SamePage

### CODE REVIEW

File: scripts/buildComponents.js
Comment (dvargasfuertes) [inline scripts/buildComponents.js:1]: Legacy script that we've already migrated

File: src/components/StaticSiteDashboard.tsx [hunk 1]
```diff
@@ -2706,6 +2708,47 @@ const RequestRedirectsContent: StageContent = ({ openPanel }) => {
...
+      inputRef.current.value = roamjsToken;
+    }
+  }, []);
+  return (
+    <div className="flex justify-between items-center">
+      <InputGroup defaultValue="" inputRef={inputRef} type="password" />
+      <Button
+        text={"Save RoamJS Token"}
+        onClick={() => {
+          try {
```
Comment (dvargasfuertes) [inline src/components/StaticSiteDashboard.tsx:2725]: hacky legacy stuff warning

File: src/components/StaticSiteDashboard.tsx [hunk 2]
```diff
@@ -2706,6 +2708,47 @@ const RequestRedirectsContent: StageContent = ({ openPanel }) => {
...
+const LegacyRoamJSToken = () => {
+  const inputRef = useRef<HTMLInputElement>(null);
+  useEffect(() => {
+    const roamjsToken = getToken();
+    if (inputRef.current && roamjsToken) {
+      inputRef.current.value = roamjsToken;
+    }
+  }, []);
+  return (
+    <div className="flex justify-between items-center">
```
Comment (dvargasfuertes) [inline src/components/StaticSiteDashboard.tsx:2720]: Ideally in the future we hide this if `window.samepage` is available. To revisit if we invest more in public website publishing

**Review by mdroidian — COMMENTED** [review]
☑

**Pull request merged**